### PR TITLE
Cp-99

### PR DESCRIPTION
**Pull request by ArsalanAhmad2000**
All changes in UI have done including customer site and admin site.

### CODE REVIEW

File: spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/admin-contact-us/admin-contact-us.component.html [hunk 1]
```diff
@@ -13,33 +13,7 @@
...
-  <div class="input-field3">
-    <div class="input-field-base">
-      <div class="input-field-base">
-        <div class="label">Email</div>
-        <input class="input1" type="text" />
-      </div>
-      <div class="hint-text3">This is a hint text to help user.</div>
-    </div>
-  </div>
+
```
Comment (Mushaf-Cyber) [inline spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/admin-contact-us/admin-contact-us.component.html:16]: why this deleted? is it your or @jshah436  code

File: spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/contact-us/contact-us.component.html [hunk 2]
```diff
@@ -64,39 +64,14 @@ <h3>Our friendly team would love to hear from you.</h3>
...
-              <div class="col-md-2 ">
-                <div class="card-body " style="margin-bottom: 2em;">
-                  <h4 class="card-title"style="margin-left:0.5rem;">KRAFT<span>CONSTRUCTIONS</span></h4>
-                  <p class="card-text "style="margin-left:0.5rem;">EMAILS</p>
-                  <p class="card-text "style="margin-left:0.5rem;">www.Kraftconstructionsco.com</p>
-                </div>
-              </div>
-            </div>
-          </div>
+
```
Comment (Mushaf-Cyber) [inline spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/contact-us/contact-us.component.html:74]: also for this as well please confirm @jshah436  then i can merge

**Review by Mushaf-Cyber — COMMENTED** [review]
please check comments

File: spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/admin-contact-us/admin-contact-us.component.html (hunk 1 above)
Comment (jshah436) [reply spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/admin-contact-us/admin-contact-us.component.html:16]: Yes, it is mine

File: spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/contact-us/contact-us.component.html (hunk 2 above)
Comment (jshah436) [reply spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/contact-us/contact-us.component.html:74]: This is also mine

Comment (ArsalanAhmad2000) [reply spring-boot-angular-15-crud-example-master/angular-15-client/src/app/components/contact-us/contact-us.component.html:74]: yes these are the new changes @Mushaf-Cyber tamara tell me to delete all these things which is not needed now.
as i said i got my last ticket from UI side for changes in whole site including admin customer both mine and junaid tickets included in that

**Review by Mushaf-Cyber — APPROVED** [review]

**Pull request merged**